packages/ember/lib/index.js

```diff
@@ -501,7 +503,9 @@ Ember.ViewUtils = {
   getViewClientRects: views.getViewClientRects,
   getViewBoundingClientRect: views.getViewBoundingClientRect,
   getRootViews: views.getRootViews,
-  getChildViews: views.getChildViews
+  getChildViews: views.getChildViews,
+  isSerializationFirstNode: isSerializationFirstNode,
+  SERIALIZATION_FIRST_NODE_STRING: SERIALIZATION_FIRST_NODE_STRING
```

Comment:
I’d rather not export this string for now (we can always come back and do it later if needed)...

Reply:
Done. :+1: